FoodBookApp/UI/Views/SpotDetail/SpotDetailViewModel.swift

```diff
@@ -12,56 +12,18 @@ import FirebaseFirestore //FIXME: delete later
-                        location: GeoPoint(latitude: 0, longitude: 0),
-                        name: "MiCaserito",
-                        price: "$",
-                        waitTime:   WaitTime(min: 5, max: 10),
-                        reviewData: ReviewData(stasts: SpotStats(cleanliness: 5, foodQuality: 4, service: 3, waitTime: 2), userReviews: [
-                            Review(
-                                content: "Lo digo y lo insisto, mi caserito es el restaurante más completo de los andes.",
-                                date: Date(),
-                                imageUrl: "https://i.ytimg.com/vi/1n6bq4wfoSU/hq720.jpg?sqp=-oaymwEXCK4FEIIDSFryq4qpAwkIARUAAIhCGAE=&rs=AOn4CLCCW-rqYpxNt3xW3Ag43ns--EwGLw",
-                                ratings: ReviewStats(cleanliness: 5, foodQuality: 5, service: 5, waitTime: 4),
-                                selectedCategories: ["Homemade", "Dessert"],
-                                title: "Me fascina!!",
-                                user: "Juan Pedro Gonzalez" // TODO: will be user uid, might need to add antoher field for email
-                            ),
-                            Review(
-                                content: "La comida me gustó pero la atención fue pésima, se demoró muchísimo, lástima.",
-                                date: Date(),
-                                imageUrl: nil,
-                                ratings: ReviewStats(cleanliness: 2,
-                                                   foodQuality: 3,
-                                                   service: 1,
-                                                   waitTime: 1),
-                                
-                                    selectedCategories: ["Poultry", "Rice", "Soup"],
-                                    title: "No lo recomiendo.",
-                                    user: "Mariana Martínez"
-                            )
-                            
-                        ]),
-                        imageLinks: nil
-                )
-
-        return spot
+    var spot: Spot = Spot(categories: [], location: GeoPoint(latitude: 0, longitude: 0), name: "", price: "", waitTime: WaitTime(min: 0, max: 5), reviewData: ReviewData(stats: SpotStats(cleanliness: 5, foodQuality: 5, service: 5, waitTime: 5), userReviews: []), imageLinks: [])
+    
+    var ratings: [String: Double] = ["Cleanliness": 0, "Waiting time": 0, "Service": 0, "Food quality": 0]
+    
+    private let repository: SpotRepository = SpotRepositoryImpl.shared
+    
+    init() {}
+    
+    @MainActor
+    func fetchSpot(spotId: String) async throws {
+        self.spot = try await repository.getSpotById(docId: spotId)
+        self.ratings = ["Cleanliness": self.spot.reviewData.stats.cleanliness / 5,"Waiting Time": self.spot.reviewData.stats.waitTime / 5,"Service": self.spot.reviewData.stats.service / 5,"Food quality": self.spot.reviewData.stats.foodQuality / 5]
```

Comment:
Ratings moved to View Model. Divides by 5 so its in the proper scale to show in the View